src/portal/src/i18n/lang/zh-cn-lang.json

```diff
@@ -334,6 +338,35 @@
         "PULL_IS_MUST" : "拉取权限默认选中且不可修改。",
         "EXPORT_TO_FILE" : "导出到文件中"
     },
+    "WEBHOOK": {
+        "EDIT_BUTTON": "EDIT",
```

Comment:
How come there is no Chinese translation?

Reply:
Because I just got the English translation from demand side.And I'm worry about the quality of translation By myself.

Reply:
You can translate first according to your own understanding. If there are problems, you can correct them later.